content/W2016/W16-media-stephanie.md

```diff
@@ -0,0 +1,16 @@
+Title: Stephanie Morillo Talk
```

Comment:
Maybe 'Stephanie Morillo: "Circuit Breaker"'? Sarah's talk title was in the page title.